### PR TITLE
Add support for multi bms and new collateral tokens

### PR DESCRIPTION
**Pull request by peroxy**

### CODE REVIEW

File: src/utils/tokenUtils.ts [hunk 1]
```diff
@@ -50,8 +60,24 @@ export const lamportsToCollateral = (lamports: Decimal, token: ScopeToken): Deci
       factor = DECIMALS_ETH;
       break;
     case 'STSOL':
+    case 'cSTSOL':
+    case 'kSOLSTSOLOrca':
+    case 'kUSHUSDCOrca':
       factor = LAMPORTS_PER_STSOL;
       break;
+    case 'cUSDC':
```
Comment (y2kappa) [inline src/utils/tokenUtils.ts:68]: we should have one const per token like here 
https://github.com/hubbleprotocol/kamino-webapp/blob/635c9f5ecd80fc179fc129cdf619ad612c3eeb34/src/utils/utils.ts#L44

Comment (peroxy) [reply src/utils/tokenUtils.ts:68]: OK, will refactor like that

Comment (y2kappa) [reply src/utils/tokenUtils.ts:68]: I think that is best model. one single source of decimals etc

**Comment by peroxy** [comment]
@y2kappa refactored the decimals code, also replaced 1_000_000 with 6 as its more easy to read

**Review by y2kappa — APPROVED** [review]

**Pull request merged**